config/google_cloud_exporter/mysql/config.yaml

```diff
@@ -14,10 +14,15 @@ processors:
     system:
       hostname_sources: ["os"]
 
-  resourceattributetransposer:
-    operations:
-      - from: host.name
-        to: agent
+  # Used for Google generic_node mapping.
+  resource:
+    attributes:
+    - key: namespace
+      value: "mysql"
+      action: upsert
+    - key: location
+      value: "us-east1-b"
```

Comment:
would `global` be more appropriate here, since we don't have a known zone to put here?

Reply:
I 100% agree with you, however, Google has instructed us to use generic_node.

I had the same question with location, their explanation was that it is where "the metrics are stored". It seems odd to me that this is a user facing parameter as it does not line up well with non GCP users.

I believe generic_node does have user facing advantages. The `namespace` parameter is used to split up metrics in the storage backend. I think this helps get around some limitations that global has.

Reply:
After further discussion, we are talking about `global` as a location value, not the global monitored resource type.